xchange-btcmarkets/src/main/java/org/knowm/xchange/btcmarkets/dto/BTCMarketsBaseResponse.java

```diff
@@ -6,9 +6,9 @@
 
 public class BTCMarketsBaseResponse {
 
-  private final Boolean success;
-  private final String errorMessage;
-  private final Integer errorCode;
+  public final Boolean success;
```

Comment:
lets use a primitive boolean - is much easier to use then a three state Boolean

Reply:
I'd have to check if it's present in all responses, null is desireable if it's not present in the response.

Reply:
if its not present when we could assume its false but ok, it might be better to do it in service code